src/backend/web/routes/opml.js

```diff
@@ -1,25 +1,34 @@
 const express = require('express');
 const opml = require('opml-generator');
+const { getFeeds, getFeed } = require('../../utils/storage');
 
 const router = express.Router();
 
 const header = {
   title: 'OPML Feeds',
-  dateCreated: new Date(2020, 2, 9),
-  ownerName: 'Jordan',
+  dateCreated: new Date(),
+  ownerName: 'Telescope',
 };
 
-const outlines = [
-  {
-    text: 'txt',
-    title: 'My Open Source Experience',
-    type: 'rss',
-    xmlUrl: 'https://jrdnlxopensource.blogspot.com/feeds/posts/default/-/open-source',
-    htmlUrl: 'https://jrdnlxopensource.blogspot.com/',
-  },
-];
+router.get('/', async (req, res) => {
+  const feedIds = await getFeeds();
+
+  const feedInfo = await Promise.all(
+    feedIds.map(feed => {
+      return getFeed(feed);
+    })
+  );
```

Comment:
I thinks could be
```js
const feedInfo = await Promise.all(feedIds.map(feed => getFeed(feed)));
```

Reply:
Good call; modified that in https://github.com/Seneca-CDOT/telescope/pull/594/commits/b785a81f48123703d91c123c998c1e0c8d7c3378